.github/workflows/check-tests.yml

```
@@ -0,0 +1,14 @@
+name: Pre-Commit Hook
```

This one is not really pre-commit, but:
```suggestion
name: Check .only flags
```